### PR TITLE
Adding Loader for EigenScape

### PR DESCRIPTION
**Pull request by iranroman**
### Description

Please include the following information at the top level docstring for the dataset's module mydataset.py:

- [x] Describe annotations included in the dataset
- [x] Indicate the total duration of the dataset in hours, and (optionally) also list the number of individual files
- [x] Mention the origin of the dataset (e.g. creator, institution)
- [x] Describe the type of audio included in the dataset
- [x] Indicate any relevant papers related to the dataset
- [x] Include a description about how the data can be accessed and the license it uses (if applicable)

#### Dataset loaders checklist:

- [x] Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file.
- [x] Run the script on the canonical version of the dataset and save the index in `soundata/indexes/` e.g. `my_dataset_index.json`.
- [x] Create a module in soundata, e.g. `soundata/my_dataset.py`
- [x] Create tests for your loader in `tests/`, e.g. `test_my_dataset.py`
- [x] Add your module to `docs/source/soundata.rst` and `docs/source/quick_reference.rst`
- [x] Run `tests/test_full_dataset.py` on your dataset.

### CODE REVIEW

**Comment by magdalenafuentes** [comment]
@iranroman thanks for submitting this PR! I'm going to take a look as soon as tests are passing. It looks like the file `TrainStation.4.wav` is missing for the tests, could you please upload it?

**Comment by iranroman** [comment]
Of course! I forgot to make the "dummy" file the one at the top of the index after my last round of debugging.

File: tests/datasets/test_eigenscape.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    default_clipid = "Beach.1"
+    dataset = eigenscape.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip(default_clipid)
+    audio_path = clip.audio_path
+    audio, sr = eigenscape.load_audio(audio_path)
+    assert sr == 48000
+    assert type(audio) is np.ndarray
+    assert len(audio.shape) == 2  # check audio is loaded as stereo
+    assert audio.shape[0] == 25  # check audio is 25ch (HOA 4th order)
+    assert audio.shape[1] == 48000 * 2.5  # Check audio duration is as expected
```
Comment (magdalenafuentes) [inline tests/datasets/test_eigenscape.py:44]: @iranroman I'm gonna merge the loader as is but I leave a note here about shortening the size of the file for testing so the testing folder stays small as we grow, especially this multichannel ones

**Review by magdalenafuentes — APPROVED** [review]
@iranroman great work! I'm merging this one :)

**Pull request merged**